webdriver/tests/execute_async_script/pinning.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from webdriver.transport import Response
+
+from tests.support.asserts import assert_error, assert_success
+
+
+def pin_script(session, name, script):
+    body = {"script": script}
+
+    return session.transport.send(
+        "POST", "/session/{session_id}/execute/pin/{script_name}".format(
+            session_id=session.session_id,
+            script_name=name),
+        body)
+
+
+def execute_async_script(session, script, name=None, args=None):
+    if args is None:
+        args = []
+    body = {"script": script, "args": args}
+
+    return session.transport.send(
+        "POST", "/session/{session_id}/execute/async/{script_name}".format(
+            session_id=session.session_id,
+            script_name=name),
+        body)
```

Comment:
You can use `session.execute_async_script()` in this test.

We only need a special “raw” implementation for the feature under test, as the session-exposed APIs come with certain conveniences, such as automatic raising of errors (instead of return-by-value) and implicit session creation.